_posts/2016-07-11-july_update.markdown

```diff
@@ -13,22 +13,22 @@ Like I wrote to [the Hydra Tech list](https://groups.google.com/forum/#!searchin
 * supports the management of top-priority content types: image-based works and generic multi-file works (think IR-like use cases)
 * offers a user interface for dynamic configuration of the application (without having to touch Ruby or YAML files)
 * provides an administrative dashboard with reporting widgets
-* allows for batch import of content and metadata
+* allows for batch import of content and metadata.
 
 Here's where our feature and application work stands:
 
 * The Hydra-in-a-Box repository application is based on Sufia 7, and we worked with the community to release several beta releases of Sufia 7 and the gems upon which it depends. A 7.0.0 release is anticipated in the next week.
 * The application is multi-tenant with each tenant having its own database tables, Solr collection, Fedora namespace, and Redis namespace.
```

Comment:
This could probably use more explanation. Why is this is a significant feature? What does it allow us to do? etc.